satpy/tests/compositor_tests/test_viirs.py

```diff
@@ -19,8 +19,10 @@
 
 import unittest
 
+import pytest
 
-class TestVIIRSComposites(unittest.TestCase):
+
+class TestVIIRSCompositesUnitTest(unittest.TestCase):
```

Comment:
Is there something here that prevents it from being converted to pytest too?

Reply:
Done. These tests were actually pretty simple. Especially when using regexes in pycharm's find/replace.